app/controllers/home_controller.rb

```diff
@@ -1,5 +1,9 @@
 class HomeController < ApplicationController
   def index
     @poems = Poem.all.order(id: :desc).includes(:user)
+    respond_to do |format|
+      format.html
+      format.json { render template: "poems/index" }
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)